### PR TITLE
Emergency Spacesuits

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

While we're on the topic of space, I'm adding emergency spacesuits to O2 lockers, this is inspired by Bee's skinsuits, which are pretty cool.

The statistics of this suit are very poor, but, when in a pinch, they'll do the job. Also, don't rub the suit up against things, it'll probably pop.

You can now find an emergency spacesuit box in all o2 lockers.

ALSO, I've removed the chance for emergency o2 lockers to just straight up not spawn, or spawn with nothing inside. I think this is a decision that I take quite seriously as it's fairly dumb and, quite annoying.

## Changelog

:cl:
add: You can now find an emergency spacesuit box in o2 lockers!
/:cl:

### CODE REVIEW

**Comment by desertmote** [comment]
Dope, what about these though? New dust-catchers?

dreamseeker_cQh6pj3FhG

File: modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+/obj/item/clothing/suit/space/emergency/examine(mob/user)
+	. = ..()
+	if(torn)
+		. += span_danger("It looks like it has been torn and is completely useless!")
+
+#undef EMERGENCY_SUIT_MIN_TEMP_PROTECT
+#undef EMERGENCY_SUIT_MAX_TEMP_PROTECT
+
+#define EMERGENCY_HELMET_MIN_TEMP_PROTECT 2.0
```
Comment (Zonespace27) [inline modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm]: why not just have these defines at the top of the file and undef them all at the bottom

File: modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+/obj/item/clothing/suit/space/emergency/update_name(updates)
+	. = ..()
+	if(torn)
+		name = "torn [src]"
+
+/obj/item/clothing/suit/space/emergency/examine(mob/user)
+	. = ..()
+	if(torn)
+		. += span_danger("It looks like it has been torn and is completely useless!")
```
Comment (Zonespace27) [inline modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm]:
```suggestion
		. += span_danger("It looks torn and useless!")
```

**Comment by Gandalf2k15** [comment]
> Dope, what about these though? New dust-catchers?
> 
> dreamseeker_cQh6pj3FhG

those are so inaccessible it's a meme

File: modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm (hunk 1 above)
Comment (Gandalf2k15) [reply modular_skyrat/modules/emergency_spacesuit/code/emergency_spacesuit.dm]: i guess

**Comment by ghost** [comment]
I thought the orange emergency suits were already in the code, even including the puncturability? The ones that spawn default in the pod emergency lockers?
Might be able to cut down on the PR and just have those in the lockers.

**Comment by Zergspower** [comment]
> 

iirc Red Alert and/or evac accessable locked like the pods

**Comment by Iamgoofball** [comment]
these are the ones goon has, right? if so :+1:

**Comment by Gandalf2k15** [comment]
> I thought the orange emergency suits were already in the code, even including the puncturability? The ones that spawn default in the pod emergency lockers? Might be able to cut down on the PR and just have those in the lockers.

They're broken

**Comment by Gandalf2k15** [comment]
> these are the ones goon has, right? if so 👍

I haven't seen goon ones, but these are pretty bad for space walking

**Comment by Gandalf2k15** [comment]
seems to work fine, mergies

**Pull request merged**